tests/helpers/Transactions/createTransaction.test.js

```diff
@@ -0,0 +1,35 @@
+const createAppointment = require("../../../helpers/Appointments/createAppointment");
```

Comment:
These imports are nightmarish. Should probably consider
https://www.npmjs.com/package/module-alias